src/common/menucmn.cpp

```diff
@@ -296,6 +296,14 @@ void wxMenuItemBase::SetAccel(wxAcceleratorEntry *accel)
     SetItemLabel(text);
 }
 
+void wxMenuItemBase::AddExtraAccel(wxAcceleratorEntry *accel)
+{
+    if ( accel )
+    {
+        m_extra_accels.push_back(accel->ToString());
```

Comment:
I wonder if we shouldn't use `ToRawString()` here to avoid any problems due to localization.